matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/helper/ThreadEventsHelper.kt

```diff
@@ -124,9 +128,38 @@ internal fun EventEntity.threadSummaryInThread(realm: Realm, rootThreadEventId:
 
     result ?: return null
 
-    return Summary(messages, result)
+    return Summary(numberOfThread, result)
 }
 
+/**
+ * Counts the number of threads in the main timeline thread summary,
+ * with respect to redactions.
+ */
+internal fun countThreads(realm: Realm, roomId: String, rootThreadEventId: String): Int? =
```

Comment:
is this counting replies in threads?

Reply:
Yes, maybe I should rename it to be more clear :)